test/new-e2e/tests/agent-metrics-logs/log-agent/linux-log/integrations/fixtures/integration.py

```diff
@@ -0,0 +1,31 @@
+from datadog_checks.base import AgentCheck
+from datadog_checks.base.utils.time import get_timestamp
+
+
+class HelloCheck(AgentCheck):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.counter = 0  # Initialize increasing variable
+
+    def check(self, instance):
+        data = {}
+        log_str = instance['log_message']
+        data['timestamp'] = get_timestamp()
+        data['ddtags'] = instance['integration_tags']
+
+        log_str = instance['log_message']
+        if instance['unique_message']:
+            log_str = instance['log_message'] * instance['log_size']
+            self.counter += 1
+            log_str = "counter: " + str(self.counter) + ' ' + log_str
+            data['message'] = log_str
+        else:
+            data['message'] = log_str
+
+        num_logs = instance['log_count']
+
+        if num_logs != 1:
```

Comment:
Can this just be 
```python
for _ in range(num_logs):
    self.send_log(data)
```

Since if `num_logs == 1`, it would stil only send 1 log?

Reply:
Good point